tests/drivers/build_all/opamp/testcase.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright 2025 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+common:
+  tags:
+    - drivers
+    - opamp
+    - userspace
+  build_only: true
+tests:
```

Comment:
Thank you first for taking the tests over. It saved me time 👍 I think the better approach to cover multiple test platforms with as less changes in .yaml as possible could be similar approach used here: https://github.com/zephyrproject-rtos/zephyr/blob/bfea9cf94f9adb38954384611968f0ae7df5b259/tests/drivers/interrupt_controller/intc_exti_stm32/testcase.yaml#L18

You may then take a look at the bash script I've used in this PR: https://github.com/zephyrproject-rtos/zephyr/pull/85508#issue-2842505399 to run twister on certain platform similar to `west twister -p <board_name> -T <test_abs_path> --vendor exp --device-testing --device-serial-baud 115200 --device-serial "/dev/ttyACM0" -v`